.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "tools-local/arcade-services"]
 	path = tools-local/arcade-services
-	url = https://github.com/dotnet/arcade-services
+	url = https://github.com/crummel/arcade-services
```

Comment:
Is this temporary, or does source-build need something from your fork?

Reply:
This was temporary. I was getting errors regarding git submodules sync for arcade services. This happened because a commit which we were referring to was in PR and not rolled out. So had to use this fork to expose that commit to sb